### PR TITLE
Extend README to contain layers information. Restructure it to follow…

### PR DESCRIPTION
**Pull request by mmusik**
… the same structure as Python README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,61 @@
...
+
+
+In this option, you will choose a SignalFx template, and then deploy a copy of the layer.
+
+1. Open your AWS console. 
+2. In the landing page, under **Compute**, click **Lambda**.
+3. Click **Create function** to create a layer with SignalFx's capabilities.
+4. Click **Browse serverless app repository**.
+5. Click **Public applications**.
+6. In the search field, enter and select **signalfx-lambda-python-wrapper**.
```
Comment (bgola-signalfx) [inline README.md]: s/python/ruby/

File: README.md [hunk 2]
```diff
@@ -14,6 +66,81 @@ gem 'signalfx-lambda'
...
+The default `SIGNALFX_METRICS_URL` points to the `us0` realm. If you are not in this realm, use correct subdomain (https://ingest.{REALM}.signalfx.com). 
+Step 2 outlines how to locate the realm.
+
+4. (Optional) Set additional environment variables for metrics configuration.
+
+Send operation timeout (in seconds) can be specified with `SIGNALFX_SEND_TIMEOUT`
+environment variable. Default value is 1 second.
+
+```bash
+   SIGNALFX_SEND_TIMEOUT=1.0
```
Comment (bgola-signalfx) [inline README.md]: integer is expected here: 1.0 --> 1

File: README.md (hunk 1 above)
Comment (mmusik) [reply README.md]: thanks. Replaced all occurences of python with ruby ;)

File: README.md (hunk 2 above)
Comment (mmusik) [reply README.md]: thanks for the catch.

**Pull request merged**